module/proto/peggy/v1/genesis.proto

```diff
@@ -34,19 +34,22 @@ message Params {
   uint64 signed_valsets_window            = 6;
   uint64 signed_batches_window            = 7;
   uint64 signed_claims_window             = 8;
-  bytes  slash_fraction_valset            = 9  [
+  uint64 target_batch_timeout_millis      = 10;
```

Comment:
Lets remove the `target_` and `_millis` from the names here. We can document that these values are in milliseconds.

Reply:
I think target is important, it's an estimate, not a precise value. Having millis or not having millis I don't really care about.